phpBB/phpbb/install/helper/iohandler/cli_iohandler.php

```diff
@@ -126,7 +126,8 @@ public function add_error_message($error_title, $error_description = false)
 		$this->io->newLine();
 
 		$message = $this->translate_message($error_title, $error_description);
-		$this->io->error($message['title'] . "\n" . $message['description']);
+		$message_string = $message['title'] . "\n" . (!empty($message['description']) ? $message['description'] : '');
```

Comment:
the EOL char is needed only when there is a description IIRC. You should move it to the ternary